.github/workflows/wheels.yaml

```diff
@@ -0,0 +1,60 @@
+name: Build CI Wheels
+
+on:
+  push:
+    branches:
+      - main
+      - stable
+    tags:
+      - yt-*
```

Comment:
am I reading right that this means anytime we push, *and* anytime we push a tag with a version, it builds?

Reply:
ah nm, I see:

> It covers both the release process and dev (building after every push to master).